src/expression/comparison_expression.cpp

```diff
@@ -80,5 +80,56 @@ const std::string ComparisonExpression::GetInfo() const {
   return os.str();
 }
 
+bool ComparisonExpression::SymmetricEquals(
+    const AbstractExpression &other) const {
+  PELOTON_ASSERT(children_.size() == 2);
+
+  if (other.GetChildrenSize() != 2) {
+    return false;
+  }
+
+  auto vcl = children_[0].get();
```

Comment:
it isn't clear exactly what `vcl` stands for. would recommend spelling it out or at least adding a comment. same applies to the other acronyms below